tests/Unit/Subscriber/Addons/CloudflareSubscriber/TestAutoPurgeByUrl.php

```diff
@@ -75,6 +75,8 @@ public function testShouldAutoPurgeByUrlWhenCacheEverythingButReturnError() {
 		$mocks = $this->getConstructorMocks( 1, '', '', '' );
 		Functions\when( 'current_user_can' )->justReturn( true );
 		Functions\when( 'get_current_user_id' )->justReturn( 1 );
+		Functions\when( 'get_rocket_i18n_home_url' )->justReturn( 'http://local.com/' );
+		Functions\when( 'get_feed_link' )->justReturn( 'http://local.com/feed/' );
```

Comment:
Using `example.org` as the placeholder domain would be better:
- `local.com` is a real URL
- `example.org` is a domain registered specifically for testing purpose
- `example.org` is the default domain used by the WP test suite for integration test